rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -22,6 +25,20 @@ pub enum NewGameGlitch {
     Yes,
     No,
 }
+#[derive(rebo::ExternalType)]
+pub enum SplitsSaveError {
+    /// There are 2 `String`s as arguments for both `CreationError` and `SaveError`.
+    /// These two strings are the filename of the file that the user attempted to save and the std::io:Error gotten from that attempt respectively.
```

Comment:
1. Way too much to read :) I was thinking more among the lines of <https://github.com/oberien/refunct-tas/blob/37224fa122bed69cc5322c01892c57b233d257f0/protocol/src/lib.rs#L21-L32>, e.g. `/// filename, io::Error.to_string()`
2. Doc comments only apply to the item directly below them. If you hover over `CreationError` in intellij, you get shown the doc, whereas hovering over `SaveError` doesn't show anything as it doesn't have a doc comment.